.github/workflows/ci.yml

```diff
@@ -0,0 +1,216 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - auto
+      - try
```

Comment:
On travis we need to include the `master` branch, because it's (well technically the repo `HEAD`) cache is used as the fallback for new branches and PRs.  That doesn't seem to apply to the caching logic on GH Actions, so I think its okay to drop it here.